.coderabbit.yaml

```diff
@@ -2,5 +2,7 @@ early_access: true
 reviews:
   path_filters:
     - "!content/blog/!*.md"
+    - "!content/blog/sync_status.yml"
```

Comment:
let's avoid to store data files from content, it should be located in data folder as I remember, please check hugo docs about this